frontend/src/metabase/ui/components/overlays/Popover/Popover.config.ts

```diff
@@ -10,7 +10,13 @@ export const popoverOverrides = {
       radius: "sm",
       shadow: "md",
       withinPortal: true,
-      middlewares: { shift: true, flip: true, size: true },
+      middlewares: {
+        shift: true,
+        flip: true,
+        size: {
+          padding: 5,
```

Comment:
fixes extra scrollbars

Reply:
can you add a comment explaining why this is needed?

Reply:
sure

Reply:
Done